charts/stable/qbittorrent/values.yaml

```diff
@@ -19,20 +25,34 @@ service:
         enabled: true
         port: 6881
         protocol: udp
+workload:
+  qbitportforward:
+    enabled: true
+    type: Deployment
+    strategy: RollingUpdate
+    replicas: 1
+    podSpec:
+      containers:
+        qbitportforward:
+          primary: true
+          enabled: true
+          imageSelector: qbitportforwardImage
+          probes:
+            liveness:
+              enabled: false
+            readiness:
+              enabled: false
+            startup:
+              enabled: false
+          env:
+            QBT_USERNAME: admin
+            QBT_PASSWORD: adminadmin
+            QBT_ADDR: http://localhost:8080
+            GTN_ADDR: http://localhost:8000
```

Comment:
Is that all the envs that it needs? 
Users can't add additional envs from the GUI (on SCALE)

Reply:
Besides that you probably need to link those envs with something like
.Values.qbitforward.username

Reply:
Yup just those 4. Second two should be universal

Reply:
note that qbitforward and qbit is not on the same network as you have defined them now.
So localhost will only be the qbitforward container

Reply:
ohh, so i need to let them set those variables? I thought the different container would be able to talk to each other. Should i add a port for gluetun and set the qbittorrent port to whatever is external in the config?

Reply:
I updated the values.yaml to use the question responses. Just need to know the best way ahead for having this sidecar reach the qbittorrent webui port and gluetun webui port.